### PR TITLE
MCKIN-21885: add filter to remove course count

### PR DESCRIPTION
**Pull request by msaqib52**

### CODE REVIEW

File: edx_solutions_organizations/views.py [hunk 1]
```diff
@@ -68,48 +74,52 @@ def list(self, request, *args, **kwargs):
...
+                q_object.add(Q(users__courseenrollment__is_active=True), Q.AND)
+
+                # annotating queryset to get number of courses
+                queryset = queryset.annotate(
+                    number_of_courses=Count(
+                        Case(
+                            When(q_object, then=F('users__courseenrollment__course_id'))
+                        ), distinct=True
+                    )
+                )
```
Comment (mudassir-hafeez) [inline edx_solutions_organizations/views.py:115]: This code is using from one endpoint of Apros where you have set this `include_course_counts `. It means this code should be removed if it is not using from Apros side.

**Review by mudassir-hafeez — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='organizations-edx-platform-extensions',
-    version='1.3.6',
+    version='1.3.7',
```
Comment (nasirhjafri) [inline setup.py]: Please do version bump when moving to Integration/Stage and after cherry-picking commits to master.

File: edx_solutions_organizations/views.py (hunk 1 above)
Comment (msaqib52) [reply edx_solutions_organizations/views.py:115]: keeping this code for now to keep the functionality intact if it is being called somewhere else, in phase 2 which will come next month we will optimise this API.

**Review by mudassir-hafeez — APPROVED** [review]

**Comment by mudassir-hafeez** [comment]
Good to go :+1:

**Pull request merged**